CHANGELOG.md

```diff
@@ -2,23 +2,44 @@
 
 ### Features
 
+### Bug Fixes
+
+### Refactoring & Build
+
+## v2.0.2 [2020-11-18]
+
+### Features
+
+1. [19979](https://github.com/influxdata/influxdb/pull/19979): Added functionality to filter task runs by time.
 1. [20036](https://github.com/influxdata/influxdb/pull/20036): Warn if V1 users are upgraded, but V1 auth wasn't enabled.
 1. [20039](https://github.com/influxdata/influxdb/pull/20039): Export 1.x CQs as part of `influxd upgrade`.
+1. [20053](https://github.com/influxdata/influxdb/pull/20053): Upgrade Flux to v0.95.0.
+1. [20058](https://github.com/influxdata/influxdb/pull/20058): UI: Upgrade flux-lsp-browser to v0.5.23.
 1. [20067](https://github.com/influxdata/influxdb/pull/20067): Add DBRP cli commands as `influxd v1 dbrp`.
 
 ### Bug Fixes
 
 1. [19987](https://github.com/influxdata/influxdb/pull/19987): Fix various typos. Thanks @kumakichi!
 1. [19991](https://github.com/influxdata/influxdb/pull/19991): Use --skip-verify flag for backup/restore CLI command.
 1. [19995](https://github.com/influxdata/influxdb/pull/19995): Don't auto-print help on influxd errors
-1. [20012](https://github.com/influxdata/influxdb/pull/20012): Validate input paths to `influxd upgrade` up-front
+1. [20008](https://github.com/influxdata/influxdb/pull/20008): Add locking during TSI iterator creation.
+1. [20012](https://github.com/influxdata/influxdb/pull/20012): Validate input paths to `influxd upgrade` up-front.
+1. [20015](https://github.com/influxdata/influxdb/pull/20015): Add same site strict flag to session cookie.
 1. [20017](https://github.com/influxdata/influxdb/pull/20017): Don't include duplicates for SHOW DATABASES
 1. [20064](https://github.com/influxdata/influxdb/pull/20064): Ensure Flux reads across all shards.
 1. [20047](https://github.com/influxdata/influxdb/pull/20047): Allow scraper to ignore insecure certificates on a target. Thanks @cmackenzie1!
 1. [20076](https://github.com/influxdata/influxdb/pull/20076): Remove internal `influxd upgrade` subcommands from help text.
 1. [20074](https://github.com/influxdata/influxdb/pull/20074): Use default DBRP mapping on V1 write when no RP is specified.
 1. [20091](https://github.com/influxdata/influxdb/pull/20091): Make the DBRP http API match the swagger spec.
 
+### Refactoring & Build
```

Comment:
What do you think about adding a section like this for changes that are mainly for the dev teams? Not sold on the title for the section, but I think it makes sense to separate them out if we want to include them in the curated log.

Reply:
cc @timhallinflux and @psteinbachs on this question, too

Reply:
We do not want expose build/refactoring changes to our users. At this point CHANGELOG has only changes in behavior (features) or/and bug fixes changes. I would not want to change this.

Reply:
:+1: will remove it